src/py42/clients/_archive_access.py

```diff
@@ -184,8 +180,8 @@ def _get_default_file_size():
 class _RestorePoller(object):
     JOB_POLLING_INTERVAL_SECONDS = 1
 
-    def __init__(self, storage_archive_client, device_guid, job_polling_interval=None):
-        self._storage_archive_client = storage_archive_client
+    def __init__(self, storage_archive_service, device_guid, job_polling_interval=None):
+        self._storage_archive_service = storage_archive_service
         self._device_guid = device_guid
         self._job_polling_interval = (
             job_polling_interval or self.JOB_POLLING_INTERVAL_SECONDS
```

Comment:
Line 192 is an example where we can use response.data now

Reply:
@alanag13 Don't want to do this?

Reply:
This is actually assigning a jobId, not getting one, so we don't need to use `data.get` to check to see if anything exists

Reply:
I don't mean data.get, I mean `size_dict = size_response.data`

Reply:
I don't see a reason why we would ever need to call `json.loads()` on a py42response's text anymore